README.md

```diff
@@ -1,250 +1,326 @@
+    your *packages lists* without having to compile everything yourself. The
+    *OpenWrt-SDK* is the fastest way to build your own packages or programs 
+    without compiling OpenWrt itself.
++ The *initrd* directory contains some initrd-images for netboot, which is
+    required on some boards to initially install OpenWrt.
+
 
 ### customizing make
 
 `make` will use by default `TARGET` and `PACKAGES_LIST_DEFAULT` defined in
 `config.mk`. You can customize this by overriding them:
 
-```
-make TARGET=mpc85xx PACKAGES_LIST_DEFAULT=backbone
-```
-in addition you can build your own image from a prebuilt imagebuilder by something like:
+``` make TARGET=mpc85xx PACKAGES_LIST_DEFAULT=backbone ```
 
-```
-make images IB_FILE=<file> TARGET=... PACKAGES_LIST_DEFAULT=...
-```
+in addition you can build your own image from a prebuilt imagebuilder by
+something like:
 
-The default target is `ar71xx-generic`. For a complete list of supported targets look in `configs/` for the target-specific configs.
-Each of these targets need a matching file in `profiles/` with the profiles (boards) that should be build with the imagebuilder.
+``` make images IB_FILE=<file> TARGET=... PACKAGES_LIST_DEFAULT=... ```
 
-additional options
+The default target is `ar71xx-generic`. For a complete list of supported targets
+look at `configs/` for the target-specific configs. Each of these targets need a
+matching file in `profiles/` with the profiles (boards) that should be build
+with the imagebuilder.
+
+#### Additional Options
 
 * IS_BUILDBOT :
-  * this will be "yes" when running on the buildbot farm and helps to save some disc-space by removing files not required anymore. On manual builds you should not set this to "yes", as you have to rebuild the whole toolchain each time.
+  * this will be "yes" when running on the buildbot farm and helps to save some
+      disc-space by removing files not required anymore. On manual builds you
+      should not set this to "yes", as you have to rebuild the whole toolchain 
+      each time.
+
 * SET_BUILDBOT :
-  * "env" the Makefile will honor the "IS_BUILDBOT" environment
-  * "yes" the Makefile will always act as "IS_BUILDBOT" was set to "yes"
-  * "no"  the Makefile will always act as "IS_BUILDBOT" was set to "no" / is unset. This way we can run builds on the buildbot like a local build.
+  * "env" the Makefile will honor the "IS_BUILDBOT"
+      environment 
+  * "yes" the Makefile will always act as "IS_BUILDBOT" was set to "yes" 
+  * "no"  the Makefile will always act as "IS_BUILDBOT" was set to "no" / is
+      unset. This way we can run builds on the buildbot like a local build.
+
+
+### Build a new router model for an existing architecture
+
+Everything that is in the [Table of Hardware](https://openwrt.org/toh/start)
+should work at least in theory. Wifi should be ath9k / ath10k. mt76 also appears
+to work. 
+
+You can use the Imagebuilder to get a list of all supported models for a
```

Comment:
Probably the better place, than the mentioned above,  to reference "make info" if we want to reference it at all.